Packages/Sites/Neos.NeosIo/Resources/Private/Templates/NodeTypes/Reference.List.html

```diff
@@ -1,3 +1,25 @@
-<div {attributes -> f:format.raw()}>
-	{references -> f:format.raw()}
+{namespace io=Neos\NeosIo\ViewHelpers}<div {attributes -> f:format.raw()}>
+	<io:simpleForm actionUri="#reference-filter" class="form form--inline" method="get" id="reference-filter">
```

Comment:
I can't find the simpleForm viewhelper, did you forget to add this one?
The div should be in a newline